Framework/Kernel/test/TimeSeriesPropertyTest.h

```diff
@@ -1345,6 +1345,47 @@ class TimeSeriesPropertyTest : public CxxTest::TestSuite {
     return;
   }
 
+  /*
+   * Test cloneWithTimeShift
+   */
+  void test_cloneWithTimeShift() {
+    TimeSeriesProperty<int> *time_series =
+        new TimeSeriesProperty<int>("IntUnixTest");
+    time_series->addValue("2019-02-10T16:17:00", 1);
+
+    TimeSeriesProperty<int> *time_series_small_shift =
+        dynamic_cast<TimeSeriesProperty<int> *>(
+            time_series->cloneWithTimeShift(100.));
+    std::vector<DateAndTime> small_shift_times =
```

Comment:
I think these can be captured by `const auto &`, as I don't think they are be deleted before the assertions